sh/tests/helm-template.bats

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ./sh/tests/libs/bats/bin/bats
```

Comment:
No need for the non-dockerized tests. It's redundant, and the script is not run directly by the user.